src/content/quickstart/_index.en.md

```diff
@@ -1,17 +1,67 @@
 +++
-title = "Quickstart"
+title = "Quickstart guide"
 date = 2020-02-19T20:03:44+01:00
 weight = 5
 pre = "<b>1. </b>"
 +++
 
+## Basic overview
 
-# Quickstart guide
+Submariner has two main core pieces (the broker and submariner), more information about 
```

Comment:
Is there a distinction between “Submariner” and “submariner” here? Are we no longer talking about the engine?

Reply:
I'm trying to simplify the explanation a little bit (as introduction).

"submariner" in this case would be "submariner-engine + submariner-route-agent [ + submariner global cidr ipam])

I'm sure it can be explained better ':)